vasquez_a10.js

```diff
@@ -0,0 +1,21 @@
+/*
+Bryan Vasquez
+vasquez_a10.js
+INFO 2124
+Thoendel
+02/23/2020
+*/
+
+
+document.getElementById('action').addEventListener('click', function() {
+    //WRITE YOUR SOLUTION BETWEEN THIS COMMENT
+    let img = document.getElementById('photo'); //Getting element by ID 
+    img.setAttribute('src', '02.jpg'); //then, changing the src for the new picture.
+
+    let h1 = document.getElementsByTagName('h1'); //Getting element by tag name
+    h1.setAttribute('class', 'greenText'); // then, adding the CSS class to the h1 tag.

```

Comment:
this isn't working since you get all the h1 tags in an array. So, you would need to h1[0].setAttribute() or put in a loop to change all the h1 tags to green text. Make sense?